web/sites/default/config/views.view.within_park_facility.yml

```diff
@@ -373,7 +315,7 @@ display:
           exclude: false
           alter:
             alter_text: true
-            text: "{% set amenities = field_park_amenities_activities|render|striptags|trim|split(\", \") %}\r\n\r\n{% if \"Dog Off-leash Area\" not in amenities and \"Fenced Dog off-leash Area\" not in amenities  and  \"No Dogs Allowed\" not in amenities %}\r\n{{ 'All dogs must be leashed in this park.'|t }}\r\n{% endif %}"
+            text: "{{ 'All dogs must be leashed in this park.'|t }}"
```

Comment:
Is the conditional logic not needed anymore?

Reply:
Yeah as you saw, I replaced the twig template logic with a taxonomy filter. This is because previously, even when the view had nothing to display, it still returned results since the filtering was done in a "Custom text" block. 

This change is in concert with all the other twig template changes you may have seen, where we now save the `drupal_block` result in a variable and use that, instead of re-calling/re-rendering it twice. TBD what kind of performance increase this will have but it definitely cleans up the code and follows best practices.

For example: https://github.com/eGovPDX/portlandor/pull/2933/files#diff-69b0d13e1089fa6da2a3bab1df4573a78abef5ffd014f7d156f640d73b062dc4L64

You'll see here that previously we had to render the view and check if the result was just the header "Park policy", meaning the custom text block was empty. Now since the logic is done using a view filter, we can simply check if length > 0.